lib/chartmogul.rb

```diff
@@ -79,10 +79,14 @@ module ChartMogul
   class << self
     extend ConfigAttributes
 
+    def global_config
+      @global_config ||= ChartMogul::Configuration.new
+    end
 
+    # This configuration is thread-safe and fits multi-account async
+    # jobs processing use case.
     def config
```

Comment:
may be rename to `thread_safe_config`

Reply:
@macharmi I thought about it, but it's accessible, so I would have to create an alias to keep backwards compatibility, maybe the comment is enough.